src/Components/Publishing/ToolTip/TooltipsDataLoader.tsx

```diff
@@ -74,18 +81,24 @@ export class TooltipsDataLoader extends Component<Props> {
 class TooltipsContextProvider extends Component<any> {
   static childContextTypes = {
     tooltipsData: PropTypes.object,
+    currentUser: PropTypes.object,
   }
 
   getChildContext() {
+    const { artists, currentUser, genes } = this.props
+
     return {
       tooltipsData: {
-        artists: this.props.artists,
-        genes: this.props.genes,
+        artists,
+        genes,
       },
+      currentUser,
     }
   }
 
   render() {
     return this.props.children
   }
 }
+
+export const TooltipsData = Artsy.ContextConsumer(TooltipsDataLoader)
```

Comment:
I wonder if just exporting `ContextConsumer(TooltipsDataLoader` as `TooltipsDataLoader` in terms of keeping the name of the module the same as the main export as you would in redux but that's just a naming preference.